packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -538,7 +538,8 @@ class HeapTreeViewState extends State<HeapTree>
                   child: createIcon(Icons.vertical_align_bottom),
                 ),
               ),
-        const SizedBox(width: defaultSpacing),
+
+        if (!controller.showTreemap.value) const SizedBox(width: defaultSpacing),
```

Comment:
Nit: move this up in the conditional starting on lin 519 that also checks controller.showTreemap.value
That code could be cleaned up by using list comprehensions instead of returning a no-op SizedBox.

Reply:
👍 sounds good - that'll be cleaner than all the ternary experssions